### ISSUE TITLE
activity suggestion box based on the weather updates of city

### ISSUE THREAD

**Issue by meghanakn473** [Mohit5Upadhyay/WeatherApplication#11]

**Comment by meghanakn473** [comment]
i can easily work on it as i have done it in my project earlier... can u please assign it to me

**Comment by Mohit5Upadhyay** [comment]
Yes, @meghanakn473 you can work on this issue to introduce suggestion box

**Comment by meghanakn473** [comment]
can you please assign it under gssoc-ext and hackoctoberfest

**Comment by meghanakn473** [comment]
could you please check my submission and assign the badges as gssoc-ext and hackoctoberfest badges

**Comment by meghanakn473** [comment]
hey @Mohit5Upadhyay can u please review my work and allow merge request and assign the gssoc-ext, hackoctober and level3 labels??
